doc/dev/structure/api.rst

```diff
@@ -0,0 +1,41 @@
+.. _`api_structure`:
+
+
+API Structure
+=============
+
+.. todo:: Authentication
+
+The API is implemented via the `django rest framework`_.
+Using `viewsets`_, most models are serialized and presented at the
+:code:`/api/` endpoint. There a browsable API is presented but also in
+json format (depending on the requested content-type or if `.json` is
+appended).
+
+View sets can be very simple, but also quite extendable. If needed,
+a lower-level approach could also be taken. However, it is recommended
+to use viewsets and other higher abstractions as much as possible to
+keep the paths consistent.
+
+Currently, we have the following paths for different models:
+
+.. code-block:: json
```

Comment:
Why is this JSON? Wouldn't a table or similar be better suited for documentation?